new-dti-website/components/products/imageCarousel.tsx

```diff
@@ -63,6 +64,20 @@ const ImageCarousel = (props: { items: carouselItem[] }) => {
           ))}
         </CarouselContent>
       </Carousel>
+      {reduceMotion && (
+        <button
+          className="absolute right-[2%] bottom-[15%] rounded-full p-2 bg-[#d63d3d] hover:bg-[#a52424] duration-300"
+          onClick={togglePlayPause}
+          aria-label={isPlaying ? 'Pause carousel' : 'Play carousel'}
+        >
+          <Image
+            src={isPlaying ? '/icons/pause.svg' : '/icons/play.svg'}
+            alt={isPlaying ? 'Pause' : 'Play'}
+            width={24}
+            height={24}
+          />
+        </button>
+      )}
```

Comment:
honestly i think we should show this play/pause icon button regardless of whether you have Reduce Motion settings toggled on or off. 

I think we should do what [apple.com](https://www.apple.com/) does at the bottom of their home page with the Apple TV/Apple Arcade/etc. carousels:
- They always show the play/pause icon button
- If you have Reduce Motion on, the carousel does _not_ animate by default
- If you have Reduce Motion off, the carousel does animate by default (but you can easily pause it)